filters/include/pcl/filters/impl/radius_outlier_removal.hpp

```diff
@@ -109,10 +109,7 @@ pcl::RadiusOutlierRemoval<PointT>::applyFilterIndices (Indices &indices)
       }
       else
       {
-        if (negative_)
-          chk_neighbors = true;
-        else
-          chk_neighbors = false;
+        chk_neighbors = static_cast<bool>(negative_);
```

Comment:
Isn't `negative_` already a bool? Why is the cast necessary?

Reply:
Not sure why `clang` felt that was necessary...